ruledata.go

```diff
@@ -716,6 +688,38 @@ func SetRules(s *NetlinkConnection, content []byte) error {
 					}
 				}
 			}
+		case "file_rules":
+			vi := v.([]interface{})
+			for ruleNo := range vi {
+				rule := vi[ruleNo].(map[string]interface{})
+				path := rule["path"]
+				if path == "" {
+					log.Fatalln("Watch option needs a path!")
+				}
+				// TODO: Add key addition support
+				// key := rule["key"]
+				perms := rule["permission"]
+				log.Println("Setting File Permissions", path, perms)
+				var dd AuditRuleData
+				dd.Buf = make([]byte, 0)
+				add := AUDIT_FILTER_EXIT
+				action := AUDIT_ALWAYS
+				_audit_syscalladded = true
+
+				err := AuditSetupWatchName(&dd, path.(string))
```

Comment:
I was mostly looking into libaudit.c for those function. 
if someone plan to build auditctl-go over libaudit-go, he should be writing those functions. 
But I also makes sense to provide extra functionality, Bonus :D

Also, I think It makes sense to change the name of AuditSetupWatchName to AuditSetupAndAddWatchDir. (Which was my original point) or just don't call AuditAddWatchDir in AuditSetupWatchName.